tests/cli/test_config.py

```diff
@@ -333,10 +333,10 @@ def test_config_command_parser():
 
     with make_temp_condarc(condarc) as rc:
         stdout, stderr, return_code = run_command(Commands.CONFIG, '--file', rc, '--add',
-                                           'disallow', 'perl')
+                                           'disallowed_packages', 'perl')
```

Comment:
So did we changed a configuration name? added an alias or ?

disallow -> disallowed_packages ?

Reply:
Changed the name, then made the old name an alias.  See https://github.com/conda/conda/pull/6932/files#diff-b48dce471b4b8aed241c3b09306aa176R131